gallery/src/pageConfigTypes.ts

```diff
@@ -6,5 +6,21 @@
  */
 import {ComponentType} from 'react';
 
+export const PAGE_TYPES = [
```

Comment:
Some of the values in this list are not used

Reply:
> Some of the values in this list are not used

They are now: https://github.com/sonatype/sonatype-react-shared-components/pull/596/commits/044591cd94cae51748267d19b8b9fd7bfc507930